app/ts/components/Transactions.tsx

```diff
@@ -24,6 +24,7 @@ function formatTransactionDescription(tx: TransactionDescription) {
 		</>
 	)
 }
+const etherScanAbiKey = 'PSW8C433Q667DVEX5BCRMGNAH9FSGFZ7Q8'
```

Comment:
Why didn't we need this before?

Reply:
we did, it was just part of the urls instead of being a variable